sites/page-details/page-details-docs-en.html

```diff
@@ -0,0 +1,289 @@
+	"language": "en",
+	"altLangPage": "page-details-docs-fr.html",
+	"breadcrumb": [
+		{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" }
+	],
+	"secondlevel": false,
+	"dateModified": "2022-04-14",
+	"share": true
+}
+---
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca site functionality</dd>
+	<dt>Last review</dt>
+	<dd>2022-04-14</dd>
+</dl>
+
+<h2>Purpose</h2>
+<p>Displays the Page details section in a footer tag.</p>
+<p>Can contain the following components:</p>
+<ul>
+	<li>Report a problem on this page</li>
+	<li>Share this page</li>
+	<li>Date modified</li>
+</ul>
+
+<h2>Working example</h2>
+<ul>
+	<li><a href="default-page-details-en.html">Default page details example</a></li>
+	<li><a href="page-details-without-buttons-en.html">Page details without Report a problem and Share this page buttons example</a></li>
+	<li><a href="page-details-without-report-btn-en.html">Page details without Report a problem button example</a></li>
+	<li><a href="page-details-without-share-btn-en.html">Page details without Share this page button example</a></li>
+</ul>
+
+<h2>How to implement</h2>
+<ul>
+	<li><p>To show the "Report a problem on this page" button, create a <code>&lt;div&gt;</code> with the following data attribute: <code>data-ajax-replace="/sites/main-footer/inc-report-problem-en.html"</code>.</p>
+		<p>For the basic HTML version, a link element should be created that points to the report a problem form page: <code>&lt;a class="btn btn-default btn-block" href="https://www.canada.ca/en/report-problem.html"&gt;Report a problem on this page&lt;/a&gt;</code></p>
+		<p>For more information about Data Ajax plugin, please visit <a href="https://wet-boew.github.io/wet-boew/docs/ref/data-ajax/data-ajax-en.html">Data Ajax documentation page</a></p>
+	</li>
+	<li><p>To show the "Share this page" button create a <code>&lt;div&gt;</code> with the following data attribute:<code>data-wb-share='{"lnkClass": "btn btn-default btn-block"}'</code> and the following CSS class <code>class="wb-share"</code></p>
+		<p>For more information about Share widget plugin, please visit <a href="https://wet-boew.github.io/wet-boew/docs/ref/share/share-en.html">Share widget page</a></p>
+	</li>
+	<li><p>The Date modified - normal text format containing the modified date.</p></li>
+</ul>
+
+<h2>Evaluation and report</h2>
+<p>There is no evaluation and report available for this component.</p>
+
+<h2>API (Version 2.0)</h2>
+<table class="table table-bordered">
+	<tr>
+		<th>CSS Class</th>
+		<th>Template</th>
+		<th>Visual rendering</th>
+		<th>Schema</th>
+	</tr>
+	<tr>
+		<td>Version 1.0</td>
+		<td>Version 2.0</td>
+		<td>Version 1.0</td>
+		<td>n.a.</td>
+	</tr>
+</table>
+
+<h3>CSS Class (v1.0)</h3>
+<p><code>pagedetails</code></p>
+
+
+<h3>Template (v2.0)</h3>
+<h4 id="tmpl-mgrt-nt">Template migration notes</h4>
+
+<section>
```

Comment:
Is there a reason to use an explicit `<section>` element here?

Reply:
implicit section used